src/application/components/Enum.js

```diff
@@ -0,0 +1,41 @@
+// @flow
```

Comment:
I think we could, if model doesn't depend on a model or enum type but took basic properties. Jerry mentioned though we probably want to separate models from enums, like the old UI does, so this separation helps for the future.